packages/perseus-editor/src/editor-page.tsx

```diff
@@ -193,13 +193,15 @@ class EditorPage extends React.Component<Props, State> {
         return issues1.concat(issues2);
     }
 
-    serialize(options?: {keepDeletedWidgets?: boolean}): any | PerseusItem {
+    serialize(options?: {keepDeletedWidgets?: boolean}): PerseusItem {
         if (this.props.jsonMode) {
             return this.state.json;
         }
-        return _.extend(this.itemEditor.current?.serialize(options), {
-            hints: this.hintsEditor.current?.serialize(options),
-        });
+        return {
+            ...this.itemEditor.current!.serialize(options),
+            hints: this.hintsEditor.current!.serialize(options) ?? [],
+            answer: undefined,
```

Comment:
**Question**: Why would we need this `answer: undefined` here?

Would it not be the same as just not adding it at all here? Or are you trying to override whatever `answer` might be in `this.itemEditor.current`? If so, why is that the case? Is this different behavior than what was here before?

Reply:
I found that I needed to provide it. I _think_ it's because it is defined as `answer: any`. So TypeScript doesn't like that it's not defined as a key, but you can assign it the value `undefined` which then essentially removes it. 🤷‍♂️ 

Ultimately, I'd like to look at removing `answer` from `PerseusRenderer` as it is a deprecated field. I searched our English content and can only find one usage of this field and it is in a "staging lifeboat" (ie, not really used). I loaded that assessment item in teh editor and the JSON view doesn't even show it, which indicates to me that the editor doesn't even acknowledge that field any longer...so I am going to remove the field from the `PerseusRenderer` type!